src/components/ImageDisplay/ImageNavigationToolbar.vue

```diff
@@ -43,20 +85,55 @@
           <span>large fits</span>
         </a>
       </p>
+      <p class="control">
+        <a class="button has-text-white is-small" 
+          @click="download_jpg">
+          <b-icon icon="download" size="is-small" />
+          <span>jpg</span>
+        </a>
+      </p>
+      <p class="control">
+        <b-button :loading="zip_download_waiting" class="button has-text-white is-small" @click="download_fits_previous_24hrs">
+          <b-icon icon="download" size="is-small" />
+          <span>last 24hrs fits</span>
+        </b-button>
+      </p>
     </b-field>
   </div>
 </template>
 
 <script>
+import axios from 'axios'
 import {mapState, mapGetters} from 'vuex';
 export default {
   name: "ImageToolbar",
   data() {
     return {
+      isDownloadModalActive: false,
+      zip_download_waiting: false,
     }
   },
 
   methods: {
+    download_fits_previous_24hrs() {
```

Comment:
The download last 24 hours button is working for me. Are you planning to include a batch download button for images older than 24 hours under the data tab?

Reply:
The downloaded folder is called uploads.zip. This is a very minor thing, but maybe a descriptive name in the future would be better.

Reply:
I'm planning on making an interface where users can select a start/end time window, and also specify whether they want only small fits, large fits, or the best available. The server supports the additional queries already so I just need to make the UI. But I wanted to throw this up so there's something in the meantime.

Reply:
And good catch on the download filename. I made a quick adjustment to `<sitename>-<download-request-timestamp>.zip` and open to other ideas if you have them. This is easy to change whenever.